README.md

```diff
@@ -1,180 +1,95 @@
-﻿# Qiskit Metal
-
-[![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.4618153.svg?style=popout-square)](https://doi.org/10.5281/zenodo.4618153) [![Release](https://img.shields.io/github/release/Qiskit/qiskit-metal.svg?style=popout-square)](https://github.com/Qiskit/qiskit-metal/releases) [![License](https://img.shields.io/github/license/Qiskit/qiskit-metal.svg?style=popout-square)](https://opensource.org/licenses/Apache-2.0) [![](https://badges.frapsoft.com/os/v2/open-source.svg?v=103)](https://github.com/Qiskit/qiskit-metal) [![](https://cdn.rawgit.com/sindresorhus/awesome/d7305f38d29fed78fa85652e3a63e154dd8e8829/media/badge.svg)](https://github.com/Qiskit/qiskit-metal)
-
-
-**Qiskit Metal** is an open-source framework for designing quantum devices.
-
-
+# Qiskit Metal 
+[![License](https://img.shields.io/github/license/Qiskit/qiskit-metal.svg?style=popout-square)](https://opensource.org/licenses/Apache-2.0)<!--- long-description-skip-begin -->[![Release](https://img.shields.io/github/release/Qiskit/qiskit-metal.svg?style=popout-square)](https://github.com/Qiskit/qiskit-metal/releases)<!--- long-description-skip-begin -->[![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.4618153.svg)](https://doi.org/10.5281/zenodo.4618153)
 >  Quantum hardware design and analysis
-
 ![Welcome to Qiskit Metal!](docs/images/zkm_banner.png 'Welcome to Qiskit Metal')
-
-### Quantum device design project
```

Comment:
we need to keep these 4 sections (intro, recorded tut, live tut, slack). But you can compact them if you like, and maybe stash them into a title that sounds something like `Community and Support`.

Reply:
Done!